include/myst/futex.h

```diff
@@ -18,12 +18,21 @@
 #define FUTEX_UNLOCK_PI      7
 #define FUTEX_TRYLOCK_PI     8
 #define FUTEX_WAIT_BITSET    9
+#define FUTEX_WAKE_BITSET    10
 #define FUTEX_PRIVATE        128
 #define FUTEX_CLOCK_REALTIME 256
 // clang-format on
 
+#define FUTEX_BITSET_MATCH_ANY 0xffffffff
+
 int myst_futex_wait(int* uaddr, int val, const struct timespec* to);
+int myst_futex_wait_ops(
```

Comment:
I wonder whether ``myst_futex_wait_bitset()`` might be a better name (since that is the only parameter that is different).

Reply:
That was the original name, until I realized that we may have to pass on more parameters for the clock type. I renamed all the operations with additional "optional" arguments to `ops` for consistency.

Reply:
But currently, the only additional parameter is ``bitset``. I think just adding additional (possibly null) parameters to ``myst_futex_wait`` generates fewer changes.